pkg/apis/submariner.io/v1/types.go

```diff
@@ -78,8 +78,13 @@ type EndpointSpec struct {
 	NATEnabled    bool              `json:"nat_enabled"`
 	Backend       string            `json:"backend"`
 	BackendConfig map[string]string `json:"backend_config,omitempty"`
+	PortConfig    map[string]string `json:"natConfig,omitempty"`
```

Comment:
To match the field name. Unless the field should be `portConfig`? (doesn't seem so)

```suggestion
	NatConfig    map[string]string `json:"natConfig,omitempty"`
```